src/GafferImage/OpenImageIOReader.cpp

```diff
@@ -523,6 +523,19 @@ void OpenImageIOReader::hashFormat( const GafferImage::ImagePlug *output, const
 	hashFileName( context, h );
 	refreshCountPlug()->hash( h );
 	missingFrameModePlug()->hash( h );
+
```

Comment:
after looking at it with Andrew, we agreed this was fixing an existing bug ( we were using the FormatPlug value in the compute but not adding it to the hash)